test/functional/ios/testapp/basics/calc-app-2-specs.js

```diff
@@ -74,9 +75,16 @@ describe('testapp - basics - calc app 2', function () {
       .log('crashlog').then(function (logsBefore) {
         numBeforeLogs = logsBefore.length;
         fs.writeFileSync(dir + '/myApp_' + Date.parse(new Date()) + '_rocksauce.crash', msg);
-      }).log('crashlog').then(function (logsAfter) {
+      })
+      .log('crashlog').then(function (logsAfter) {
         logsAfter.length.should.be.above(0);
-        logsAfter.length.should.not.equal(numBeforeLogs);
+        if (env.IOS83) {
+          // getting the crash logs in 8.3 consumes them, so we will only get the one added above
```

Comment:
interesting, what mechanism retrieves the log for us under the hood? weird that the log is sensitive to something reading data from it.